lib/algoliasearch-rails.rb

```diff
@@ -162,7 +166,11 @@ def attributes_to_hash(attributes, object)
 
     def get_attributes(object)
       attributes = if @attributes.nil? || @attributes.length == 0
-        get_default_attributes(object)
+        if @serializer.nil?
+          get_default_attributes(object)
+        else
+          {}
+        end
```

Comment:
I don't understand this change @julienbourdeau; also, it doesn't match https://github.com/algolia/algoliasearch-rails/pull/266/files#diff-dfaec67e907148ea8211e17340f0f30cL145